try to make site more intuitive

pabl0rg opened the pull request:
Hi Cedric,

I am trying to remember what I found confusing about Kobalt initially.  I still have some doubts that I will try to use to improve the site.  For now, I made the js more flexible, reorganized the plugin writing section, and tried to highlight the main benefits of switching to Kobalt on the homepage.

I consider this site a good example for us: http://www.tinylog.org/

tropikoder committed:
try to make site more intuitive

tropikoder committed:
move github link to contributing page

tropikoder committed:
improve getting-started

cbeust commented:
There are a lot of formatting changes that cloud the actual changes.

Do you think you could send another PR with just the text changes?

If not, just summarize where you think the doc was confusing and how to improve it?

Thanks!

pabl0rg commented:
I used Sublime's html formatter to clean up the html format which was
somewhat messy.  That may have been a mistake.

If you check out my fork, you can browse the pages on your machine.  I
removed and added a minimal amount of text.  I mostly organized what seemed
to be misplaced information, such as the license and slack information in
the Documentation page.  I created a Getting Started.

Best,
Juan

On Tue, Jan 5, 2016 at 4:17 PM, Cedric Beust notifications@github.com
wrote:

> There are a lot of formatting changes that cloud the actual changes.
> 
> Do you think you could send another PR with just the text changes?
> 
> If not, just summarize where you think the doc was confusing and how to
> improve it?
> 
> Thanks!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cbeust/kobalt-doc/pull/7#issuecomment-169151423.

cbeust commented:
I spent some time reviewing all the changes and I like them all, thanks for doing this!

I'm going to do a few touch ups after the merge but go ahead if you want to make additional changes (e.g. removing the jumbotrons).

Thanks!
